lagunita/lms/static/sass/_overrides.scss

```diff
@@ -216,8 +216,8 @@ body.discussion {
 }
 
 #seq_content {
-    font-size: 16px !important;
-    letter-spacing: normal !important;
```

Comment:
If this "just works", why did you add the `!important` in the first place?

Reply:
it was insurance...